examples/directions/components/directions/directions.tsx

```diff
@@ -0,0 +1,47 @@
+import {useEffect} from 'react';
+import {useDirections, useGoogleMap} from '@ubilabs/google-maps-react-hooks';
+
+const DirectionsExample = () => {
+  const {map} = useGoogleMap();
+
+  const directionsOptions = {
+    renderOnMap: true,
+    renderOptions: {
+      suppressMarkers: true,
+      polylineOptions: {strokeColor: '#FB2576', strokeWeight: 4}
+    }
+  };
+
+  // Use findAndRenderRoute to get directions and render a route to the map
+  const {findAndRenderRoute} = useDirections(directionsOptions);
+
+  useEffect(() => {
+    if (!map || !findAndRenderRoute) {
+      return;
+    }
+
+    // Form Request to pass to findAndRenderRoute
```

Comment:
Maybe you can add a link to the [Google Documentation for Directions Requests](https://developers.google.com/maps/documentation/javascript/directions#DirectionsRequests) here. Or is it better in the README? Or both?
```suggestion
    // Directions Request to pass to findAndRenderRoute
    // https://developers.google.com/maps/documentation/javascript/directions#DirectionsRequests
```

Reply:
I think I will put a link to the docs here and when I start the refactoring, I'll probably rewrite some part of the README and add it there as well. Okay? :)